src/mixins/name-request-mixin.ts

```diff
@@ -17,7 +17,12 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
    * @param email the applicant's email address
    * @returns the name request response payload
    */
-  async validateNameRequest (nrNumber: string, phone?: string, email?: string): Promise<NrResponseIF> {
+  async validateNameRequest (
+    nrNumber: string,
+    phone?: string,
+    email?: string,
+    nrType?: CorrectionTypes
```

Comment:
This isn't the NR type. It's the type of company name correction being done.

Ref: https://github.com/bcgov/business-edit-ui/blob/main/src/enums/correctionTypes.ts

I think this method/mixin shouldn't care about the type of correction being done. It should only focus on NR. Therefore, I think it would be better if you passed in the information you really care about: is this is correction with a new NR?

Reply:
Actually, is this method called in any scenario other than for a name change NR? It's not clear to me.